pkgs/development/python-modules/spectral-cube/default.nix

```diff
@@ -25,7 +25,7 @@
 
 buildPythonPackage rec {
   pname = "spectral-cube";
-  version = "0.6.6-unstable-2025-06-11";
+  version = "0.6.6";
```

Comment:
The `# tag =  "v${version}";` line needs below to be uncommented. Also the `rev` line needs to be removed, and the `hash` needs to be updated.